databricks/koalas/plot/core.py

```diff
@@ -176,128 +176,26 @@ def _get_plot_backend(backend=None):
         KoalasPlotAccessor._backends[backend] = module
         return module
 
-    @staticmethod
-    def _format_args(backend_name, data, kind, kwargs):
-        """
-        Format the arguments.
-        Assigns default values to plotting functions.
-        Maps the argument to the appropriate backend.
-        """
-        from databricks.koalas import DataFrame, Series
-
-        data_preprocessor_map = {
-            "pie": TopNPlot().get_top_n,
-            "bar": TopNPlot().get_top_n,
-            "barh": TopNPlot().get_top_n,
-            "scatter": TopNPlot().get_top_n,
-            "area": SampledPlot().get_sampled,
-            "line": SampledPlot().get_sampled,
-        }
-        # make the arguments values of matplotlib compatible with that of plotting backend
-        args_map = {
-            "plotly": {
-                "logx": "log_x",
-                "logy": "log_y",
-                "xlim": "range_x",
-                "ylim": "range_y",
-                "yerr": "error_y",
-                "xerr": "error_x",
-            }
-        }
-
-        if isinstance(data, Series):
-            positional_args = {
-                ("ax", None),
```

Comment:
I think this mapping was copied from pandas, which is actually not needed in our case since we're dispatching it in each method e.g.) `KoalasPlotAccessor.area`.